apps/cookbook/src/app/app-routing.module.ts

```diff
@@ -7,6 +7,7 @@ import { routes } from './app.routes';
   imports: [
     RouterModule.forRoot(routes, {
       scrollPositionRestoration: 'enabled',
+      relativeLinkResolution: 'legacy',
```

Comment:
What's the impact of this? 🤔

Reply:
I'm not sure. It was added by the migration tool.

https://angular.io/api/router/ExtraOptions#relativeLinkResolution says

> The default in v11 is `corrected`

so it's a bit odd that it's configured to `legacy`... 🤷‍♂️